### PR TITLE
Cross build to sbt 1.0 and 0.13

### PR DESCRIPTION
**Pull request by andresilva**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 lazy val root = (project in file(".")).settings(
   sbtPlugin := true,
+  crossSbtVersions := Vector("0.13.16", "1.0.2"),
   name := "sbt-diff-project",
   licenses += "MIT" -> url("https://raw.githubusercontent.com/opt-tech/sbt-diff-project/master/LICENSE"),
   addSbtPlugin("com.mayreh" % "sbt-reverse-dependency" % "0.1.1"),
```
Comment (andresilva) [inline build.sbt]: I need to update this plugin once https://github.com/ocadaruma/sbt-reverse-dependency/pull/1 is merged and a new version is released.

**Comment by andresilva** [comment]
Could you please release a new version after this is merged? I'm currently using this plugin at work (thanks for the fast CI builds! 👍) and this is the only plugin blocking our update to sbt 1.0.

**Comment by andresilva** [comment]
This still needs to be updated. sbt's `Load` API is now private, so this won't compile.

**Comment by andresilva** [comment]
I have updated this PR and I think it's ready to be reviewed.

File: circle.yml [hunk 2]
```diff
@@ -4,4 +4,4 @@ dependencies:
 
 test:
   override:
-    - sbt scripted
+    - sbt ^scripted
```
Comment (ocadaruma) [inline circle.yml]: I think `^scripted` should be quoted.
`- sbt '^scripted'`

Comment (andresilva) [reply circle.yml]: Fixed.

**Comment by ocadaruma** [comment]
LGTM
@andrebeat Thanks a lot!

**Pull request merged**